ecc/bls12381/ff/cyclo12.go

```diff
@@ -0,0 +1,72 @@
+package ff
+
+import "fmt"
+
+// Cyclo12 represents an element of the 12-th cyclotomic group.
+type Cyclo12 [2]Fp6
```

Comment:
Eh, not when you expose Add and Sub. That makes it into a ring.

Reply:
of course :)